pagopa/io-app
# chore: [IOPID-516] FL deploy workflow

## Short description
This PR adda a way do deploy a FL beta version of the app.

## How to test
- Create a tag with `fl` instead of `rc` (e.g. `1.2.3-fl.0`).  
- The `release-fl.yml` should start instead of `release.yml`.
- For **Android** it should upload the release build to the close channel `Beta - Login Veloce`, while for **iOS** it should add to the TestFlight build the group `Beta - LV`
